dashboard/config/scripts/levels/New Sprite Lab Project.level

```diff
@@ -487,4 +487,4 @@
       </xml>
     </toolbox_blocks>
   </blocks>
-</GamelabJr>
\ No newline at end of file
+</GamelabJr>
```

Comment:
intentional?

Reply:
Nope -- that's my editor. I'll revert that